### PR TITLE
Fix: Front End connection to BE for rating after success checkout order

### PR DESCRIPTION
**Pull request by luthfiazhari**
### JIRA Link
[https://blocksphere2020.atlassian.net/browse/DBIO-538](url)

### Changelog / Description
Set Endpoint to backend for store feedback from customer

### CODE REVIEW

File: src/store/feedback/index.js [hunk 1]
```diff
@@ -15,6 +21,21 @@ export default {
...
       commit('SET_FEEDBACK', feedback)
+      const data = {
+        "lab_id": feedback.lab_id,
+        "service_id": feedback.service_id,
+        "order_id": feedback.orderId,
+        "rating_by": feedback.customer_id,
+        "rating": feedback.rate,
+        "created": feedback.createdAt
+    }
+      axios.post(`${baseUrl}/rating`, data)
```
Comment (agustinustheo) [inline src/store/feedback/index.js:32]: You are already inside an `async` function. I think it would be better to use `await` here. I would usually let this slide but knowing that this is code on the Vuex store I prefer it to be tidy. @luthfiazhari 

And please do not use `console.log` if it is not necessary.

File: src/views/Dashboard/Customer/OrderHistoryDetail/index.vue [hunk 2]
```diff
@@ -409,6 +411,8 @@ export default {
       this.orderId = this.$route.params.number;
       this.order = await getOrdersData(this.api, this.orderId);
       this.orderDetail = await getOrdersDetail(this.api, this.orderId);
+      console.log(this.order, 'this.order')
```
Comment (agustinustheo) [inline src/views/Dashboard/Customer/OrderHistoryDetail/index.vue:414]: Do you need this? Maybe we need to make a Debug mode that allows `console.log`? cc: @kevinjanada

Comment (agustinustheo) [reply src/views/Dashboard/Customer/OrderHistoryDetail/index.vue:414]: Making a Debug mode is **optional** and is just a suggestion.

**Review by agustinustheo — CHANGES_REQUESTED** [review]
TLDR; Please fix the code on the Vuex store. Don't use `.then` inside `async` functions.

**Review by agustinustheo — APPROVED** [review]
